nav2_mppi_controller/include/nav2_mppi_controller/tools/utils.hpp

```diff
@@ -276,8 +265,12 @@ inline bool withinPositionGoalTolerance(
 template<typename T>
 auto normalize_angles(const T & angles)
 {
-  auto theta = xt::eval(xt::fmod(angles + M_PIF, 2.0f * M_PIF));
-  return xt::eval(xt::where(theta < 0.0f, theta + M_PIF, theta - M_PIF));
+  return (angles + M_PIF).unaryExpr([&](const float x){
```

Comment:
```suggestion
  return (angles + M_PIF).unaryExpr([&](const float x) {
```

Seems like a linting issue - I'm surprised if CI doesn't pick up on that